scripts/chassismodulesshow

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```

Comment:
can we move the functions into the show/config show/chassis_module.py file?

Reply:
Done and added UT.